test/Starcounter.Linq.Tests/LinqTests.cs

```diff
@@ -127,19 +130,23 @@ public void TypeIs_NestedTwice()
             Assert.Equal("SELECT E FROM \"Starcounter\".\"Linq\".\"Tests\".\"Employee\" E WHERE ((E.\"Department\".\"Company\" IS \"Starcounter\".\"Linq\".\"Tests\".\"Company\")) FETCH 1",
                 Sql(() => Objects<Employee>().FirstOrDefault(d => d.Department.Company is Company)));
         }
+        // ReSharper restore IsExpressionAlwaysTrue
 
+        // ReSharper disable IsExpressionAlwaysFalse
         [Fact]
-        public void TypeIsNot()
+        public void TypeIs_ReservedWordType()
```

Comment:
Please remove the `_` usage, by our own guides we do not recommend using `_` in C#.

Reply:
Are you sure that this rules should be used everywhere? For me it is useful to use `_` in test names for highlighting some aspects.
For instance, I have `WhereStringEqual_Take_FirstEnumEqual` test which shows a sequence of operations which the method tests. If I have to replace underscores, it will looks like `WhereStringEqualThenTakeThenFirstEnumEqual` which totally unreadable.

Reply:
Not sure :), and you are right, in tests it actually makes sense.